src/extension/statusbartoggler.ts

```diff
@@ -2,18 +2,24 @@ import { Disposable, StatusBarItem, window } from "vscode";
 import { Config } from "./config";
 
 export class StatusBarToggler implements Disposable {
-    private static readonly coverageText = "Coverage";
-
-    private static readonly loadingText = ["$(loading~spin)", StatusBarToggler.coverageText].join(" ");
+    private static readonly loadingText = ["$(loading~spin)", "Coverage"].join(
+        " "
+    );
 
     private static readonly idleIcon = "$(circle-large-outline)";
 
-    private static readonly watchCommand = "coverage-gutters.watchCoverageAndVisibleEditors";
-    private static readonly watchText = [StatusBarToggler.idleIcon, "Watch"].join(" ");
-    private static readonly watchToolTip = "Coverage Gutters: Click to watch workspace.";
+    private static readonly watchCommand =
+        "coverage-gutters.watchCoverageAndVisibleEditors";
+    private static readonly watchText = [
+        StatusBarToggler.idleIcon,
+        "Watch",
+    ].join(" ");
+    private static readonly watchToolTip =
+        "Coverage Gutters: Click to watch workspace.";
```

Comment:
Random thought (not needed for this PR), I guess these should be translated one day for i18n purposes 🤔.

@mattseddon any exposure / experience with translating vscode extension strings?